rollouts/__init__.py

```diff
@@ -10,7 +10,20 @@
 
 # Eventually we want all repos to use this
 # This flag will just help us with the rollout process
-USE_LABEL_INDEX_IN_REPORT_PROCESSING_BY_REPO_ID = Feature(
-    "use_label_index_in_report_processing",
-    0.0,
-)
+# USE_LABEL_INDEX_IN_REPORT_PROCESSING_BY_REPO_ID = Feature(
+#     "use_label_index_in_report_processing",
+#     0.0,
+# )
+
+
+class TempFeature:
+    def __init__(self):
+        pass
+
+    def check_value(self, id, default=False):
+        if id == 16621196 or id == 16273544:
```

Comment:
gio's overrides

Reply:
lolol, perhaps would be useful to a comment in the code as to what this is